File: java_work/ex0707/src/ex0707/ex02.java
package ex0707;

import zoo.Duck;

public class ex02 {
		ex02(){ //생성자
		zoo.Dog dog = new zoo.Dog();
		zoo.Cat cat = new zoo.Cat();
		Duck duck = new Duck();
		sound(dog,cat,duck);
		}
		//함수
		public void sound(zoo.Dog dog,zoo.Cat cat,Duck duck){
			dog.sound();
			cat.sound();
			duck.sound();
		}
		public static void main(String[] args) {
			new ex02();//메인에서 생성 
		}
}

File: python_work/220707/ex02.py
from zoo.Dog import Dog as d
from zoo.Cat import Cat
from zoo.Duck import Duck
class ex02():
    def __init__(self):
        self.dog=d() 
        self.cat=Cat()
        self.duck=Duck()
        # self.sound()
    def sound(self):
        self.dog.sound()
        self.cat.sound()
        self.duck.sound()
    

ez=ex02()
ez.sound()